ui/shared/Revision.jsx

```diff
@@ -94,13 +126,34 @@ export class Revision extends React.PureComponent {
         </span>
         <AuthorInitials title={`${name}: ${email}`} author={name} />
         <span
-          title={comment}
+          ref={this.spanRef}
           className={`ml-2 revision-comment overflow-hidden text-nowrap ${commentColor}`}
+          id={`revision${revision}`}
         >
           <em>
-            <BugLinkify>{comment}</BugLinkify>
+            <BugLinkify id={revision}>{comment}</BugLinkify>
           </em>
         </span>
+        {ready && (
+          <Tooltip
+            isOpen={tooltipOpen}
+            toggle={() => this.handleTooltip()}
+            target={`revision${revision}`}
+            innerClassName="tooltip-content"
```

Comment:
Let's add `placement="top-start"` as well.  Otherwise, if the width of the comments for a push are of uneven lengths, the tooltips have odd placement.